File: MODUL 3 PEMROGRAMAN 1/PRAK302-2310817120002-SITI RATNA DWINTA SARI.py
for i in range(5):
    nilai = int(input())
    if nilai >= 80:
        print("A")
    elif 70 <= nilai < 79:  
        print("B")
    elif 60 <= nilai < 69: 
        print("C")
    elif 50 <= nilai < 59:  
        print("D")
    else:
        print("E")

File: MODUL 3 PEMROGRAMAN 1/PRAK302-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
int main (){
  int nilai;
   for (int i = 0; i < 5; i++){
    scanf("%d", &nilai);
    if (nilai >= 80){ 
        printf("A\n");
    } 
    else if (nilai >= 70 && nilai < 79){
        printf("B\n");
    }
    else if (nilai >= 60 && nilai < 69){
        printf("C\n");
    }
    else if (nilai >= 50 && nilai < 59){
        printf("D\n");
    }
    else if (nilai <= 50){
        printf("E\n");
    }
   } return 0;
}